src/Theme/Space.stories.tsx

```diff
@@ -13,13 +18,7 @@ export const spaces = [
   { size: '64px', name: 'xxlarge' },
 ];
 
-<Meta title="Theme|Space" />
-
-# Spacing
-
-This is the spacing that will be used throught all edozo apps (customer facing). The design files can be found [here](https://www.figma.com/file/OpCqZzeuBbb8NBshcsBRpc/Edozo-maps-DS?node-id=155%3A1235).
-
-<Story name="Default">
+export const Space: Story = () => (
   <DisplayBoxWrapper>
     {spaces.map(space => (
       <DisplayBox size={space.size} borderRadius="xsmall">
```

Comment:
I get a typescript warning here too
![Uploading Screenshot 2020-10-06 at 11.46.21.png…]()

Reply:
Not sure what is wrong with the screenshot in GH but it might be fixed now?